tests/test_parser.py

```diff
@@ -49,14 +49,14 @@
 from inmanta.ast.variables import AttributeReference, Reference
 from inmanta.execute.util import NoneValue
 from inmanta.parser import ParserException, SyntaxDeprecationWarning
-from inmanta.parser.plyInmantaParser import parse
+from inmanta.parser.plyInmantaParser import myparse
 
 
 def parse_code(model_code: str):
     root_ns = Namespace("__root__")
     main_ns = Namespace("__config__")
     main_ns.parent = root_ns
-    statements = parse(main_ns, "test", model_code)
+    statements = myparse(main_ns, "test", model_code)
```

Comment:
This is a bit weird in my opinion. As I understand it the only difference between `parse` and `my_parse` is the caching. Perhaps naming them `parse_cached` and `parse` respectively would be better? Or even just `parse` as we had previously with an optional `use_cache: bool` parameter.

Reply:
That would be better. I'll update it.

Reply:
(I didn't rename parse, because it is used in many places, if I can keep the API stable, I prefer that)

Reply:
I agree

Reply:
but there may be a beter name for the underlying parse functionality